cmd/util_test.go

```diff
@@ -394,7 +394,7 @@ func TestMakeSubsitutions(t *testing.T) {
 	}
 	for _, c := range cases {
 		t.Run(c.input, func(t *testing.T) {
-			s := makeSubstitutions(c.input, "bin", c.subst)
+			s, _ := makeSubstitutions(c.input, "bin", c.subst)
```

Comment:
Would be nice to test that the substitutions returned are as expected

Reply:
Yup, totally agree I'm writing tests at the moment apologise for the slight delay should get them pushed tonight.